packages/cookie-consent/cookie-consent.scss

```
@@ -0,0 +1,88 @@
+@import "~@fremtind/jkl-core/variables/all";
+@import "~@fremtind/jkl-core/mixins/all";
+@import "~@fremtind/jkl-button/button.scss";
+@import "~@fremtind/jkl-list/list.scss";
+
+$card-corner-radius: rem(2px);
+$container-width: rem(660px);
+$container-width--compact: rem(320px);
+
+.jkl-cookie-consent-modal + backdrop {
+    position: fixed;
+    background: rgba(0, 0, 0, 0.3);
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    transition: 0.2s;
+    z-index: $z-index--modal !important;
+}
+.jkl-cookie-consent-modal[hidden],
+.jkl-cookie-consent-modal + backdrop[hidden] {
+    pointer-events: none;
+    visibility: hidden;
+    display: block;
+    opacity: 0;
+}
+
+.jkl-cookie-consent-modal {
+    z-index: $z-index--floating !important;
+    position: absolute;
+    margin: auto;
+    top: 15%;
```

Kanskje bruke fixed her også? Kan fort ende opp med å komme langt ned på siden sånn det er nå. Må i så fall ha med en `max-height` og `overflow-y: auto`. Testet ut med dette, og det så ganske smooth ut:
```suggestion
    position: fixed;
    margin: auto;
    top: 10%;
    max-height: 80vh;
    overflow-y: auto;
    box-sizing: border-box;
```

`box-sizing: border-box` bør nok uansett med. Slik det er nå flyter dialogen ut av siden på mobil :/

---

Veldig fin løsning! Scrollen knakk litt, men konkluderte med at jeg ikke trenger flexbox og da ordna ting seg 👍